mithril-aggregator/src/services/epoch_service.rs

```diff
@@ -100,9 +110,17 @@ pub trait EpochService: Sync + Send {
 
     /// Get the [SignedEntityConfig] for the current epoch.
     fn signed_entity_config(&self) -> StdResult<&SignedEntityConfig>;
+
+    /// Get the total number of SPOs for the current epoch.
+    fn total_spo(&self) -> StdResult<&u32>;
+
+    /// Get the total stake for the current epoch.
+    fn total_stake(&self) -> StdResult<&u64>;
 }
 
 struct EpochData {
+    cardano_era: String,
```

Comment:
This could have its own type alias `CardanoEra`

Reply:
Done. ✅